modules/oe_theme_contact_forms/README.md

```diff
@@ -0,0 +1,14 @@
+# OpenEuropa Contact Forms companion module
+
+This module is a theming companion module to the [OpenEuropa Contact Forms](https://github.com/openeuropa/oe_contact_forms) module.
+It provides the logic needed to theme the corporate contact forms.
+
+## Installation
+
+Make sure you you have read the OpenEuropa Contact Forms's [README.md](https://github.com/openeuropa/oe_contact_forms/blob/master/README.md)
+before enabling this module.
+
+After enabling this module make sure you assign the following permissions to the anonymous user role, so visitors can
+correctly access corporate contact forms.
+
+- `Access corporate contact form`
```

Comment:
I created a corporate contact form with alias, and I can access the form as anonymous user without this permission.
![image](https://user-images.githubusercontent.com/22004498/85007417-cc8aa500-b15b-11ea-87ce-ab767fe72e35.png)

Reply:
Please check. If it's true, there is an issue in oe_contact_forms.

Reply:
fixed

Reply:
This permission is also needed "Use the site-wide contact form"